emmo/ontology.py

```diff
@@ -150,6 +151,18 @@ def __dir__(self):
         if self._dir_name:
             s.update(e.name for e in lst if hasattr(e, 'name'))
 
+        # updateing with namespace keys
+        # Attention: What to do if a prefLabel coincides with a namespace key?
+        if 'namespaces' not in self.__dict_:
```

Comment:
Nice. We repeat this several places. Probably a good idea to factor it out into an own method.

Reply:
An alternative would be to make 'namespaces' a property. Then 'namespaces' would always be there and behave as an attribute, but will under the hood be created the first time it is accessed.

Reply:
> An alternative would be to make 'namespaces' a property. Then 'namespaces' would always be there and behave as an attribute, but will under the hood be created the first time it is accessed.

NOt sure I follow.
like @property and def namespace(